tests/flow/test_generic_cmds.py

```diff
@@ -185,6 +186,10 @@ def test_copy_compressed():
     env = Env()
     env.skipOnCluster()
     with env.getClusterConnectionIfNeeded() as r:
+        res = r.execute_command('INFO')
```

Comment:
@OfirMos can we wrap this into a function and put in `includes`?

Reply:
I've added a function to RLtest